draft/examples/suggest-properties-response/valid/example.json

```diff
@@ -3,12 +3,17 @@
     {
       "name": "coordinate location",
       "description": "geocoordinates of the subject. For Earth, please note that only WGS84 coordinating system is supported at the moment",
-      "id": "P625"
+      "id": "P625",
+      "match_qualifiers": [
+        {"id": "ExactMatch", "name": "Exact match of the coordinates"},
+        {"id": "DecimalPlaces-N", "name": "Match the coordinates with a precision of N decimal places"}
+      ]
     },
     {
       "name": "place of birth",
       "description": "most specific known (e.g. city instead of country, or hospital instead of city) birth location of a person, animal or fictional character",
-      "id": "P19"
+      "id": "P19",
+      "match_qualifiers": ["schema:containsPlace", "schema:containedInPlace"]
```

Comment:
I think we should settle on using only one format for the `match_qualifiers` field, otherwise it's painful to parse for clients. I'd go for the format with objects (which makes it possible to display human-readable info and is more extensible).

Also, the corresponding JSON schema should be updated accordingly (now that we are close to merging this, I think)

Reply:
Implemented in https://github.com/reconciliation-api/specs/pull/131/commits/c3c220a535805cac8ac75e6a90b503a5af8d7431 & https://github.com/reconciliation-api/specs/pull/131/commits/fb606689307067b26ed0fdbcd232130aa71d6599.